Lib/asyncio/base_events.py

```diff
@@ -393,6 +393,8 @@ def create_task(self, coro):
                 del task._source_traceback[-1]
         else:
             task = self._task_factory(self, coro)
+            tasks._set_task_name(task, name)
```

Comment:
This line should be unindented or you should pass the name to `task.Task(..)` few lines above.  Please add a test that Tasks get custom names when created via `loop.create_task(name=..)` with a standard factory.